base/cvd/cuttlefish/host/commands/cvd/fetch/fetch_cvd_parser.cc

```diff
@@ -130,6 +130,9 @@ std::vector<Flag> GetFlagsVector(FetchFlags& fetch_flags,
   flags.emplace_back(
       GflagsCompatFlag("otatools_build", vector_flags.otatools_build)
           .Help("source for the host ota tools"));
+  flags.emplace_back(
+      GflagsCompatFlag("chrome_os_build", vector_flags.chrome_os_build)
+          .Help("source for a ChromeOS build"));
```

Comment:
We should give the user some hint of the format for this flag, that it can be a single value or a triple separated by slashes.

Reply:
Also, `kUsageMessage` above is no longer accurate about the format of `*_build` flags with this addition.  Perhaps we update with the format of this flag in that message instead of adding it here?  Your call either way, but we should note the exception.

Reply:
Done